recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-vdevices.cfg

```diff
@@ -6,6 +6,8 @@ vgsx = [ 'backend=DomD, osid=1' ]
 
 # Android uses disk with it's own partitions so no index after xvda
 disk = [ 'backend=DomD, phy:/var/run/android-disks/doma, xvda' ]
+#vdisk = [ 'backend=DomD, disks=rw:/var/run/android-disks/doma' ]
```

Comment:
as i understand, for the case of virtio support, we need to uncomment the code, probably new config like doma-vdevices-virtio.cfg is better solution ?